classes/bodyText/BodyTextFile.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * @file classes/bodyText/BodyTextFile.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2003-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class BodyTextFile
+ *
+ * @brief BodyTextFile file class.
+ */
+
+namespace PKP\bodyText;
+
+use APP\facades\Repo;
+use PKP\submissionFile\exceptions\UnableToCreateFileContentException;
+use PKP\submissionFile\SubmissionFile;
+
+class BodyTextFile
+{
+    public ?string $loadingContentError = null;
+    public ?string $bodyTextContent = null;
+    public bool $isDefaultContent = true;
+
+    public function __construct(
+        public int $publicationId,
+        public ?int $submissionId = null,
```

Comment:
The `$submissionId` appears to be unused. (We can derive it, when needed, from the `$publicationId` anyway -- and I prefer if we avoid redundant and possibly conflicting IDs if we can.)